Add Session "What's new in HealthKit"

atereshkov opened the pull request:
Hi there!

I tried to do it myself and I found out really tough to make it real, thank you guys! Appreciate your hard work!

atereshkov committed:
Add Session "What's new in HealthKit"

atereshkov committed:
Updated url

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+
+- **Symptoms**
+  - Developers can now track symptoms in HealthKit, read and write symptom samples
+  - HealthKit attempted to cover and track a wide range of symptoms (shortness of breath, sleep changes, appetite changes, fever, headache, fever etc)
```
```suggestion
  - HealthKit attempted to cover and track a wide range of symptoms (shortness of breath, sleep changes, appetite changes, fever, headache, etc)
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
-
+https://developer.apple.com/wwdc20/10182/
+
+Presenters: _Netra Kenkarey_
```
```suggestion
Presenter: _Netra Kenkarey_
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+- **Electrocardiogram (ECG)**
+  - ECG samples will be available for reading in the latest version of iOS and watchOS 
+  - An ECG sample can be read as an [HKElectrocardiogram](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram) (it represents a waveform as a series of voltage values)
+  - The [HKElectrocardiogram](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram) sample is a collection of voltage measurements
```
This is already said in the line above 😄 sounds a bit redundant
```suggestion
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+    - Apple Watch will give the result of the recording in the form of a classification
+    - The classification is divided into 2 types
+      - Sinus Rhithm (heart is beating in a uniform pattern)
+      - Atrial fibrillation (form of irregular rhythm, user shopuld probably go see their doctor)
```
```suggestion
      - Atrial fibrillation (form of irregular rhythm, user should probably go see their doctor)
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+  - [symptomStatus (HKElectrocardiogram.SymptomsStatus)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551984-symptomsstatus)
+    - It tells if the user associated a symptom with this ECG (e.g. heartburn, tightness in the chest)
+    - The symptom experienced can be recorded along with the ECG
+  - [averageHeartRage (HKQuantity?)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551980-averageheartrate)
```
Is it possible to format it like this? (not sure if it will display correctly)

```suggestion
  - [averageHeartRage (`HKQuantity?`)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551980-averageheartrate)
```


If so, I'd recommend adding the backticks to all the Swift types above (e.g. `HKElectrocardiogram.Classification`, `HKElectrocardiogram.SymptomsStatus`, `HKQuantity?`, `Int`, etc) 🙏

atereshkov replied:
It works! :)

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+  - [samplingFrequence (HKQuantity?)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551983-samplingfrequency)
+  - [numberOfVoltageMeasurements (Int)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551982-numberofvoltagemeasurements)
+    - It refers to the individual voltage measurements that make up an ECG sample
+- **[HKEelectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)**
```
```suggestion
- **[HKElectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)**
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+  - [numberOfVoltageMeasurements (Int)](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551982-numberofvoltagemeasurements)
+    - It refers to the individual voltage measurements that make up an ECG sample
+- **[HKEelectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)**
+  - To retrieve the individual measurements run the [HKEelectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)
```
```suggestion
  - To retrieve the individual measurements run the [HKElectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+    - It refers to the individual voltage measurements that make up an ECG sample
+- **[HKEelectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)**
+  - To retrieve the individual measurements run the [HKEelectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery)
+  - Fetch the ECG samples with any of the HealthKit queries and then initialize the [HKEelectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery) with the fetched sample
```
```suggestion
  - Fetch the ECG samples with any of the HealthKit queries and then initialize the [HKElectrocardiogramQuery](https://developer.apple.com/documentation/healthkit/hkelectrocardiogramquery) with the fetched sample
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+- **Mobility**
+  - **New set of mobility types** (available for reading and writing on the latest iOS and watchOS)
+    - Walking speed and step length
+    - Walking assymetry and double sypport percentage
```
```suggestion
    - Walking assymetry and double support percentage
```

rogerluan commented on README.md:
```diff
@@ -952,13 +952,43 @@ Presenter: _Daniel Miao_
+    - Walking speed and step length
+    - Walking assymetry and double sypport percentage
+    - Stair ascent and descent speed
+    - Six minute walk test distance
```
```suggestion
    - Six-minute walk test distance
```

rogerluan reviewed (changes requested):
Great summary! 

There're some minor typos here and there, but overall, it's well written!

Thanks for your contribution 💪 it's the community that empowers this repo ❤️

atereshkov committed:
Apply suggestions from code review

Fixed some typos and applied suggestions

Co-authored-by: Roger Oba <rogerluan.oba@gmail.com>

atereshkov committed:
Added backticks

atereshkov commented:
@rogerluan thank you buddy for all the suggestion and nice finds! I had no chance today to write it in a text editor and did it in a default git markdown so a bunch of typos here. I'll take care next time :)

Btw, I've fixed all of them and also added the backticks. They're displaying correctly with the related urls. Cool! :)

rogerluan reviewed (approved):
> @rogerluan thank you buddy for all the suggestion and nice finds! I had no chance today to write it in a text editor and did it in a default git markdown so a bunch of typos here. I'll take care next time :)
> 
> Btw, I've fixed all of them and also added the backticks. They're displaying correctly with the related urls. Cool! :)


Perfect!! Thanks 🚀

Blackjacx commented:
@atereshkov okay if you want and have time you can follow the [contribution guide](https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md) and create a new PR. When you're quick it will be the first one to try out the brand new readme generation 🥳

Already looking forward to it :)

atereshkov commented:
Sure @Blackjacx, I'll close this one than.

stherold commented:
Hope it's not too complicated to follow the new pattern. If you don't have the time, I can integrate it for you - really it's no problem ^^
